tests/Keycloak/Repositories/EloquentKeycloakClientRepositoryTest.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Keycloak\Repositories;
+
+use App\Keycloak\Client;
+use App\Keycloak\Realm;
+use App\Keycloak\RealmCollection;
+use App\Keycloak\Repositories\EloquentKeycloakClientRepository;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Ramsey\Uuid\Uuid;
+use Tests\TestCase;
+
+final class EloquentKeycloakClientRepositoryTest extends TestCase
+{
```

Comment:
A test for an update is missing

Reply:
No longer needed because we don't do updates anymore